sections/06_discussion.md

```diff
@@ -54,46 +54,52 @@ For domain-specific models, we previously described approaches for the
 model was able to produce accurate diagnoses in which the contribution of
 previous hospital visits could be directly interpreted. Choi et al
-[@tag:Choi2016_gram] later extended this work to take into account the structure
-of disease ontologies and found that the concepts represented by the model were
-aligned with medical knowledge. Che et al. [@tag:Che2015_distill] introduced a
+later extended this work [@tag:Choi2016_gram]  to take into account the
+structure
+of disease ontologies, finding that the concepts represented by the model were
+aligned with medical knowledge. Finally, Che et al. [@tag:Che2015_distill]
+introduced a
 knowledge-distillation approach which used gradient boosted trees to learn
 interpretable healthcare features from trained deep models.
 
 ### Data limitations
 
-A lack of large-scale, high-quality, labeled training data has impacted deep
+A lack of large-scale, high-quality, correctly labeled training data has
+impacted deep
 learning in nearly all applications we have discussed, from healthcare to
 genomics to drug discovery.  The challenges of training complex, high-parameter
 neural networks from few examples are obvious, but uncertainty in the labels of
-training instances can be just as problematic.  In genomics, for example,
-labeled data may be derived from an experimental assay that has known and
+those examples can be just as problematic.  For example, in genomics
+labeled data may be derived from an experimental assay with known and
 unknown technical artifacts, biases, and error profiles.  It is possible to
 weight training examples or construct Bayesian models to account for uncertainty
-or non-independence in the data. For example, Park et al.
+or non-independence in the data. To this end, Park et al.
 [@doi:10.1371/journal.pcbi.1002957] estimated shared non-biological signal
 between datasets to correct for non-independence related to assay platform or
 other factors in a Bayesian integration of many datasets. However, such
-techniques are rarely placed front and center in the description of methods, so
-these steps may be overlooked.
+techniques are rarely placed front and center in any description of methods,
+and so may be easily overlooked.
 
 For some types of data, especially images, it is straightforward to augment
-training datasets by splitting one labeled example into multiple examples. An
+training datasets by splitting a single labeled example into multiple examples.
+For example, an
 image can easily be rotated, flipped, or translated and retain its label
 [@doi:10.1101/095794].  3D MRI and 4D fMRI (with time as a dimension) data can
-be decomposed into sets of 2D images [@doi:10.1101/070441]. This greatly expands
-the number of training examples in domains where only hundreds of subjects are
-available but artificially treats images from the same volume as independent
+be decomposed into sets of 2D images [@doi:10.1101/070441]. This can greatly
+expand
+the number of training examples
```

Comment:
I thought it was implicit: the idea is to compensate for a small training set. So, reiterating that the set was small just lengthens the text and makes it more labourious to read.